nominees/open-terms-archive.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Open Terms Archive",
+  "aliases": [
+    ""
+  ],
+  "description": "Open Terms Archive tracks and makes visible changes to the Terms Of Service of the main online services.",
+  "website": "https://opentermsarchive.org/",
+  "license": [
+    {
+      "spdx": "EUPL-1.2",
```

Comment:
@nathanbaleeta, why did you change the license in this commit when the repo clearly states that their license is a [properly accepted data](https://github.com/unicef/publicgoods-candidates/blob/main/docs/licenses.md#data) license `ODC-By-1.0`? The CI error may be a capitalization error, as the license is valid as [ODC-By-1.0](https://github.com/unicef/publicgoods-candidates/blob/9582750b80379535fb115a6d813375de40dcc2e0/nominee-schema.json#L150) but not `ODC-BY-1.0' as originally submitted.

Reply:
Also, I noticed an extra trailing space in the listing of that particular license in `nominee-schema.js`, for which I proposed a fix in #642. Please merge that before making changed on this PR, as it may affect the CI.

Reply:
@lacabra the initial license submitted was `EUPL-1.2` which is an open source software license. On visiting the `[licenseURL](https://github.com/ambanum/OpenTermsArchive-snapshots#license)` they submitted, I discovered a clause at the bottom of the page which clearly indicates that data in their repository is licensed under `ODC-BY 1.0` license, hence reason for changing. Thanks.